agnpy/spectra/spectra.py

```diff
@@ -250,6 +288,9 @@ def evaluate(gamma, k, p, gamma_min, gamma_max):
     def __call__(self, gamma):
         return self.evaluate(gamma, self.k, self.p, self.gamma_min, self.gamma_max)
 
+    def gamma_values(self):
+        return np.logspace(np.log10(self.gamma_min), np.log10(self.gamma_max), 200)
```

Comment:
this 200 would be better as an optional parameter

Reply:
this function turned out not to be really needed, so i removed it